graphpype/utils_tests.py

```diff
@@ -0,0 +1,39 @@
+"""
+Support function for loading test datasets
+"""
+import os
+
+
+def load_test_data(name):
```

Comment:
There should be a docstring for any public function you add :)

Reply:
added